Search/BinarySearch.js

```diff
@@ -48,8 +48,23 @@ function binarySearchIterative (arr, x, low = 0, high = arr.length - 1) {
   // if low > high => we have searched the whole array without finding the item
   return -1
 }
+function binarySearchIterativeAlternative (arr, x) {
+  let low = -1
+  let high = arr.length
+  while (low < high - 1) {
+    const half = Math.floor((low + high) / 2)
```

Comment:
This is a minor simplification over what the other implementation does. Consider opening a separate PR, changing the other implementation to `const mid = Math.floor((low + high) / 2)`.